web/html/src/manager/systems/delete-system.tsx

```diff
@@ -33,8 +33,8 @@ class DeleteSystem extends React.Component<Props, State> {
   }
 
   handleDelete = cleanupErr => {
-    const nocleanupParam = cleanupErr ? { nocleanup: "true" } : {};
-    return Network.post(`/rhn/manager/api/systems/${this.props.serverId}/delete`, $.param(nocleanupParam))
+    const nocleanupParam = cleanupErr ? { nocleanup: "true" } : { nocleanup: "false" };
+    return Network.post(`/rhn/manager/api/systems/${this.props.serverId}/delete`, nocleanupParam)
```

Comment:
Nitpick:

Since @Etheryte's recent improvements, you can pass proper data objects directly to `Network` methods:

```suggestion
  handleDelete = (cleanupErr: boolean) => {
    return Network.post(`/rhn/manager/api/systems/${this.props.serverId}/delete`, { nocleanup: cleanupErr })
```

Reply:
That's a good point. Thanks I applied the suggested changes